### ISSUE TITLE
Example of multi output Kriging on the fire satellite model: paramers are not optimized

### ISSUE THREAD

**Issue by josephmure** [openturns/openturns#2544]
### What happened?

![image](https://github.com/openturns/openturns/assets/31989332/473c0e65-22d7-4efa-829d-9f2126369312)

In http://openturns.github.io/openturns/1.22/auto_meta_modeling/kriging_metamodel/plot_kriging_multioutput_firesatellite.html : no optimization is actually done because optimization is disabled. Is this behavior desired?
@m-balesdent @l-brevault 

### How to reproduce the issue?

```shell
Just run the script plot_kriging_multioutpu_firesatellite.py
```

### Version

1.22

### Operating System

Linux

### Installation media

conda

**Comment by m-balesdent** [comment]
No it is not, this has been fixed in PR #2547

**Comment by sofianehaddad** [comment]
I disabled the optimization in #2212  because we got huge differences between local & CI results
Now should be ok as @jschueller fixed the issue on CI some months ago
